src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -75,4 +75,8 @@
     justify-content: space-between;
     align-items: center;
   }
+
+  .btnActive {
+    color: blue;
+  }
```

Comment:
This blue color doesn't fit overall design. Could you propose something better? ;) 

On the other hand, don't use named colors. Prioritize HEX values over them.